### PR TITLE
[AIRFLOW-155] Documentation of Qubole Operator

### PR DESCRIPTION
**Pull request by msumit**
Dear Airflow Maintainers,

Please accept this PR that addresses the following issues:
- _https://issues.apache.org/jira/browse/AIRFLOW-155_

Thanks,
Sumit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ Currently **officially** using Airflow:
 * [Lucid](http://luc.id) [[@jbrownlucid](https://github.com/jbrownlucid) & [@kkourtchikov](https://github.com/kkourtchikov)]
 * [Lyft](https://www.lyft.com/)[[@SaurabhBajaj](https://github.com/SaurabhBajaj)]
 * [Nerdwallet](https://www.nerdwallet.com)
+* [Qubole](https://qubole.com)[[@msumit](https://github.com/msumit)]
```
Comment (criccomini) [inline README.md]: Nit: a space between the company and person (you) would be good:

```
[Qubole](https://qubole.com) [[@msumit](https://github.com/msumit)]
```

File: airflow/contrib/operators/qubole_operator.py [hunk 2]
```diff
@@ -5,96 +5,82 @@
 
 class QuboleOperator(BaseOperator):
     """
-    Executes commands on Qubole (https://qubole.com).
+    Execute tasks (commands) on QDS (https://qubole.com).
+
+    :param qubole_conn_id: Connection id which consists of qds auth_token
```
Comment (criccomini) [inline airflow/contrib/operators/qubole_operator.py:10]: This stuff should all be a docstring under the **init** method.

**Comment by criccomini** [comment]
The formatting for the kwargs looks non-standard to me. I'm not entirely sure how Sphinx handles kwargs documentation (http://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html). I poked around a bit, but couldn't find any examples. If you can't, I'm OK with moving forward as is. Just let me know.

File: airflow/contrib/operators/qubole_operator.py (hunk 2 above)
Comment (msumit) [inline airflow/contrib/operators/qubole_operator.py:10]: I'd it inside `init` only, but docs was not reflecting them. Infact all other operators from contrib which having docstring inside `init` not showing up. Operators like `SSHExecuteOperator` or `HipChatAPIOperator` have docstring right after class definition and being reflected. Not sure if something wrong with my setup only.

**Comment by msumit** [comment]
@criccomini I found from this [link](https://pythonhosted.org/an_example_pypi_project/sphinx.html#function-definitions) that you can use `Args` or `Kargs` in docstrings.

**Comment by msumit** [comment]
screen shot 2016-06-01 at 6 48 17 pm

File: airflow/contrib/operators/qubole_operator.py (hunk 2 above)
Comment (criccomini) [inline airflow/contrib/operators/qubole_operator.py:10]: K, let's leave as-is then.

**Comment by criccomini** [comment]
@msumit if you fix the spacing nit, I can merge this

**Comment by msumit** [comment]
@criccomini added a space.

**Pull request closed without merging**